include/spm/evt_snd.h

```diff
@@ -6,80 +6,82 @@
 CPP_WRAPPER(spm::evt_snd)
 
 // evt_snd_bgmon(s32 bgmPlayerId, const char * name)
-EVT_DECLARE_USER_FUNC(evt_snd_bgmon, 2)
+EVT_DECLARE_USER_FUNC(evt_snd_bgmon, 2);
 
-EVT_UNKNOWN_USER_FUNC(evt_snd_bgmon_f_d)
+EVT_DECLARE_USER_FUNC(evt_snd_bgmon_f_d, 3);
 
 // evt_snd_bgmoff(s32 bgmPlayerId)
-EVT_DECLARE_USER_FUNC(evt_snd_bgmoff, 1)
+EVT_DECLARE_USER_FUNC(evt_snd_bgmoff, 1);
+EVT_DECLARE_USER_FUNC(evt_snd_bgmoff_f_d, 2);
 
-EVT_UNKNOWN_USER_FUNC(evt_snd_bgmoff_f_d)
-EVT_UNKNOWN_USER_FUNC(func_800d2268)
-EVT_UNKNOWN_USER_FUNC(func_800d2294)
-EVT_UNKNOWN_USER_FUNC(func_800d22d8)
+EVT_UNKNOWN_USER_FUNC(func_800d2268);
+EVT_UNKNOWN_USER_FUNC(func_800d2294);
+EVT_UNKNOWN_USER_FUNC(func_800d22d8);
 UNKNOWN_FUNCTION(func_800d231c);
 UNKNOWN_FUNCTION(func_800d2388);
 UNKNOWN_FUNCTION(func_800d23cc);
 UNKNOWN_FUNCTION(func_800d2438);
 UNKNOWN_FUNCTION(func_800d247c);
 
 // evt_snd_get_bgm_wait_time(s32 bgmPlayerId, s32& ret)
-EVT_DECLARE_USER_FUNC(evt_snd_get_bgm_wait_time, 2)
+EVT_DECLARE_USER_FUNC(evt_snd_get_bgm_wait_time, 2);
 
 UNKNOWN_FUNCTION(evt_snd_get_bgm_name);
 
 // evt_snd_sfxon(const char * name)
 EVT_DECLARE_USER_FUNC(evt_snd_sfxon, 1)
 
+
 UNKNOWN_FUNCTION(func_800d2834);
 UNKNOWN_FUNCTION(func_800d2894);
-UNKNOWN_FUNCTION(evt_snd_sfxon_character);
+
+EVT_DECLARE_USER_FUNC(evt_snd_sfxon_character, 4);
 
 // evt_snd_sfxon_3d(const char * name, f32 x, f32 y, f32 z)
-EVT_DECLARE_USER_FUNC(evt_snd_sfxon_3d, 4)
+EVT_DECLARE_USER_FUNC(evt_snd_sfxon_3d, 4);
 
 UNKNOWN_FUNCTION(func_800d2a58);
 
 // evt_snd_sfxon_npc(const char * sfxName, const char * npcName)
-EVT_DECLARE_USER_FUNC(evt_snd_sfxon_npc, 2)
+EVT_DECLARE_USER_FUNC(evt_snd_sfxon_npc, -1);
```

Comment:
Why did you change this? The function definitely takes 2 args

Reply:
I have no idea honestly, although I added the semicolon for consistency